<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EngineConfigDialog</class>
 <widget class="QWidget" name="EngineConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
     <property name="buddy">
      <cstring>nameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KLineEdit" name="nameLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="commandLabel">
     <property name="text">
      <string>&amp;Command</string>
     </property>
     <property name="buddy">
      <cstring>commandLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="protocolLabel">
     <property name="text">
      <string>&amp;Protocol</string>
     </property>
     <property name="buddy">
      <cstring>protocolComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KComboBox" name="protocolComboBox">
     <item>
      <property name="text">
       <string>XBoard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UCI</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KLineEdit" name="commandLineEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
